bundles/config/org.eclipse.smarthome.config.core/src/main/java/org/eclipse/smarthome/config/core/ConfigDescriptionRegistry.java

```diff
@@ -149,9 +170,18 @@ protected void removeConfigDescriptionProvider(ConfigDescriptionProvider configD
         List<ConfigDescriptionParameter> parameters = new ArrayList<ConfigDescriptionParameter>();
         List<ConfigDescriptionParameterGroup> parameterGroups = new ArrayList<ConfigDescriptionParameterGroup>();
 
+        URI aliasedURI = uri;
+        for (ConfigDescriptionAliasHandler aliasHandler : configDescriptionAliasHandlers) {
```

Comment:
I'd think this code must be moved further below. The alias should only kick in as a fallback, if no other provider was able to provide a specific description. (request for thing -> no thing desc found -> therefore using thing-type desc of thing).